statsmodels/tsa/stattools.py

```diff
@@ -574,6 +574,65 @@ def pacf_yw(x, nlags=40, method='unbiased'):
     return np.array(pacf)
 
 
+def pacf_burg(x, nlags=None, demean=True):
```

Comment:
This name (`pacf_burg`) has a different structure from `levinson_durbin_pacf`, but of course they do different things w.r.t. the pacf. Just wanted to check that this was desired.

Reply:
The most comparable is `pacf_yw` which has signature `def pacf_yw(x, nlags=40, method='unbiased')` vs `def pacf_burg(x, nlags=None, demean=True):`

There is no sense of alternative methods here so that parameter makes no sense. `nlags=None` is the future of acf/pacf estimations (I have another PR that fixes this), and then `10*log10(nobs)`.

As for `levinson_durbin` vs `lebinson_durbin_pacf`, the interpretation of `x` depending another input is IMO a mistake. It should have one job -- convert acovf to arcoefs (and other stuff).  This makes it clear that the user can choose their preferred pacf estimator before running LD, and LD_pacf doesn't need to know about all of the pacf estimators available.